src/frontend/hooks/useHentKlagebehandlinger.ts

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { useHttp } from '@navikt/familie-http';
+
+import { hentKlagebehandlinger } from '../api/hentKlagebehandlinger';
+
+export const HentKlagebehandlingerQueryKeyFactory = {
+    fagsak: (fagsakId: number | undefined) => ['klagebehandlinger', fagsakId],
```

Comment:
Tror ikke denne trenger å ta i mot `undefined`

```suggestion
    fagsak: (fagsakId: number) => ['klagebehandlinger', fagsakId],
```